<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploaderConfigWidget</class>
 <widget class="QWidget" name="UploaderConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Uploader configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="settings">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="commonSettings">
      <attribute name="title">
       <string>Common settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labDefaultHost">
           <property name="text">
            <string>Default image host</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbxDefaultHost"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkAutoCopyMainLink">
         <property name="text">
          <string>Always copy the link to the clipboard</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>165</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="hostsSettings">
      <attribute name="title">
       <string>Hosts settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="labHosts">
           <property name="text">
            <string>Settings for: </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbxHosts"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedHosts"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
